docs/www/nodule-grid.js

```diff
@@ -377,6 +377,368 @@
+                        "Lung-RADS assessment": "Lung-rads 0",
+                        "Lesion Review Status": "Accepted",
+                        "Maximum 2D diameter": "5.5",
+                        "Maximum 3D diameter": "6.2",
+                        "Maximum perpendicular 2D diameter": "4.2",
+                        "Mean 2D diameter": "4.9",
+                        "Volume": "34.5"
+                    },
+                    {
+                        "Tracking Identifier": "R9",
+                        "Tracking Unique Identifier": "1.3.12.2.1107.5.8.21.192168059.30000021092410281563200001347",
+                        "Finding": "Pulmonary nodule",
+                        "Attenuation Characteristic": "Solid",
+                        "Radiographic Lesion Margin": "Lesion with circumscribed margin",
+                        "Finding site": "Upper lobe of right lung",
+                        "Lung-RADS assessment": "Lung-rads 2",
+                        "Lesion Review Status": "Accepted",
+                        "Maximum 2D diameter": "5.4",
+                        "Maximum 3D diameter": "5.4",
+                        "Maximum perpendicular 2D diameter": "4.0",
+                        "Mean 2D diameter": "4.7",
+                        "Volume": "21.4"
+                    }
+                ];
+
+                //VAPALS.displayNotification("Updating Data");
+                if (!confirm("Do you really want to do the data update?")) {
+                    return;
+                }
+
+                // if number of nodules shown is less than the number of entries in theData array then increase the number of nodules to they match
+                while (settings.getNoduleCount() < theData.length) {
+                    _addNodule()
+                }
+
+                noduleId = 1;
+                theData.forEach(obj => {
+                    Object.entries(obj).forEach(([key, value]) => {
+                        // Need to work: “Finding”: “Pulmonary nodule”,
+
+                        // Need to work: “Finding site”: “Upper lobe of right lung”
+                        //So assuming using the "Most likely location" field ( cect1ll ) of the form.
+                        //“Upper lobe of right lung” = "RUL"
+                        //"Upper lobe of left lung" = "LUL"
+                        //"Lower lobe of right lung" = "RLL"
+                        //"Lower lobe of left lung" = "LLL"
+                        if (key === "Finding site") {
+                            ll = "#cect" + noduleId + "ll";
+                            if (!$(ll).hasClass("update-data")) {
+                                $(ll).addClass("update-data");
+                                $(ll).attr("original-value", $(ll).val());
+                                $("<button class=revert-field title='Revert back to the original data for this field' onclick=revertField(this)><i class='fa fa-arrow-circle-left'></i></button>").insertAfter(ll);
```

Comment:
I think the title should include the value that it would revert back to. For dropdown fields, use the text of the option, not the code as this would be useless to the user. Please also use proper HTML with quotes around each value of each attribute. You will need to escape them in this command. 

As I mentioned earlier, the revertField(this) is not sustainable. Instead pass the ID. If you follow my prior suggestions, this should end up being `revertField(fieldSelector)`